src/otx/core/data/module.py

```diff
@@ -63,18 +63,15 @@ def __init__(
         auto_num_workers: bool = False,
         device: DeviceType = DeviceType.auto,
         input_size: int | tuple[int, int] | None = None,
+        adaptive_input_size: Literal["auto", "downscale", "none"] = "none",
```

Comment:
```suggestion
        input_size_adapter: Literal["auto", "downscale", "none"] = "none",
```
Might be consistent w/ `input_size_multiplier`

Reply:
IMHO, adaptive_input_size looks more suitable for me. how about leaving it as it is?

Reply:
It looked like "input size adaptation method or criteria" to me so that I suggested the "adapter". Feel free to keep it or not :)